### PR TITLE
UI upgrade initial commit

### PR DESCRIPTION
**Pull request by ambud**
Creating brand new Dr. Kafka UI. 

NOTE: This PR doesn't attach the new UI and doesn't connect the build system.

### CODE REVIEW

File: drkafka/src/main/java/com/pinterest/doctorkafka/api/TopicsApi.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  }
+
+  @GET
+  public Set<Topic> getTopics(@PathParam("clusterName") String clusterName) {
+    KafkaClusterManager cm = checkAndGetClusterManager(clusterName);
+    KafkaState currentState = cm.getCurrentState();
+    Map<String, Set<TopicPartition>> topicPartitions = currentState.getKafkaCluster()
+        .getTopicPartitions();
+    Set<Topic> collect = topicPartitions.entrySet().stream()
+        .map(e -> new Topic(e.getKey(), clusterName, e.getValue().size(), 3, 0,
```
Comment (kabochya) [inline drkafka/src/main/java/com/pinterest/doctorkafka/api/TopicsApi.java]: Are the urp and replica count part still TODOs?

File: drkafka/src/main/resources/webapp2/public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```
Comment (kabochya) [inline drkafka/src/main/resources/webapp2/public/manifest.json]: Change this?

File: drkafka/src/main/java/com/pinterest/doctorkafka/api/TopicsApi.java (hunk 1 above)
Comment (ambud) [reply drkafka/src/main/java/com/pinterest/doctorkafka/api/TopicsApi.java]: yes

File: drkafka/src/main/resources/webapp2/public/manifest.json (hunk 2 above)
Comment (ambud) [reply drkafka/src/main/resources/webapp2/public/manifest.json]: fixed

File: drkafka/src/main/java/com/pinterest/doctorkafka/api/TopicsApi.java (hunk 1 above)
Comment (ambud) [reply drkafka/src/main/java/com/pinterest/doctorkafka/api/TopicsApi.java]: Will fix in second iteration once we have the state info available right now setting it to 0.

**Pull request merged**